src/containers/Questions/questions.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Button, message, Input, Progress, Icon, Popconfirm } from 'antd';
+import schema from './questionValidation';
+import './questions.css';
+
+const { TextArea } = Input;
+
+const entryData = [
+  {
+    id: 1,
+    heading: 'Today, Iam grateful for',
+    percent: 34,
+  },
+  {
+    id: 2,
+    heading: 'Challenge I face',
+    percent: 68,
+  },
+  {
+    id: 3,
+    heading: 'Iam looking to develop',
+    percent: 100,
+  },
+];
+
+class Questions extends React.Component {
+  constructor(props) {
```

Comment:
We are using es7 in our code style and we don't have constructors in this style

Reply:
I know , but the rules for unused constructor haven't merged yet , so Iam waiting to overcome conflict.